tools/protovalidate-conformance/internal/cases/cases_strings.go

```diff
@@ -650,7 +720,7 @@ func stringSuite() suites.Suite {
 		"well_known_regex/header_name/strict/invalid/empty": {
 			Message: &cases.StringHttpHeaderName{Val: ""},
 			Expected: results.Violations(
-				&validate.Violation{FieldPath: "val", ConstraintId: "string.well_known_regex.header_name"}),
```

Comment:
Do we see this as a breaking change? Empty values used to fail with IDs like `header_name` and `uri` but now they fail with `header_name_empty` and `uri_empty` and etc. I suppose this is probably OK since protovalidate is still in beta.

Reply:
Yes this is fine.